package.json

```diff
@@ -10,10 +10,7 @@
     "setupTestFrameworkScriptFile": "./enzyme_config.js"
   },
   "lint-staged": {
-    "*.{js,jsx,json,css,scss}": [
-      "prettier --write",
-      "git add"
-    ],
+    "*.{js,jsx,json,css,scss}": ["prettier --write", "git add"],
```

Comment:
you can probably revert this right?

Reply:
This formatting is done somewhere on a post commit... i did not include this file, so i have no way to undo it.
Keeps going back and forth... need to look into it.